php/7.3/base/apache.Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM php:7.3-apache-buster
+
+MAINTAINER Daniel Rose <daniel-rose@gmx.de>
+
+ENV TERM xterm
+ENV COMPOSER_MEMORY_LIMIT -1
+ENV PATH_TO_JELLYFISH = /var/www/jellyfish/releases/current
+
+RUN set -ex; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+        vim \
+        nano \
+        curl \
+        wget \
+        locales \
+        zsh; \
+    rm -rf /var/lib/apt/lists/*; \
+    \
+    echo "en_US.UTF-8 UTF-8" > /etc/locale.gen; \
+    echo "de_DE.UTF-8 UTF-8" >> /etc/locale.gen; \
+    locale-gen
+
+# composer
+RUN set -ex; \
+    \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+        git \
+        zip \
+        libzip-dev; \
+    rm -rf /var/lib/apt/lists/*; \
+    \
+    docker-php-ext-configure zip --with-libzip; \
+    docker-php-ext-install zip; \
+    \
+    wget https://raw.githubusercontent.com/composer/getcomposer.org/master/web/installer -O /tmp/installer; \
+    php /tmp/installer --no-ansi --install-dir=/usr/bin --filename=composer --quiet; \
+    rm /tmp/installer; \
+    composer self-update
```

Comment:
Is there a reason to add empty lines here?

Reply:
Better to read?!